### PR TITLE
Switch to prezto community fork

### PR DESCRIPTION
**Pull request by paulmelnikow**
Hi there, thanks for this useful tool, and for supporting prezto out of the box. That project made a community fork a couple months ago, which recently joined up with.

This is a first pass at switching over to the community fork. I haven't used Antigen before, so bear with me. I need to test this. I also wanted to open it up for comments.

Any advice on how to get the tests to run, or to go about writing new tests?

By the way, Antigen seems like such a better way to manage plugins than the centralized route taken by oh-my-zsh/prezto/zim. I'm interested in thinking through how to move things in that direction with prezto.

### CODE REVIEW

**Comment by desyncr** [comment]
> Any advice on how to get the tests to run, or to go about writing new tests?

You'll need[ `cram`](https://bitheap.org/cram/). New tests should go in `tests/use.t`. Use mocks to avoid touching the network, as in [here](https://github.com/zsh-users/antigen/blob/develop/tests/use.t#L2).

Thanks for the contribution. Gonna review this tonight.

**Comment by paulmelnikow** [comment]
Thanks, got the test to run. After looking at the other tests and the note in `tests/use.t`,  I'm not sure it's feasible to write an automatic test.

I tested this manually by running `antigen use prezto` with the latest release, then installing this, and running `antigen use prezto && antigen update`. Also with a clean install.

File: bin/antigen.zsh [hunk 1]
```diff
@@ -364,6 +375,8 @@ antigen () {
       branch="${url#*|}"
     fi
     install_or_update=true
+    # Update remote if needed.
+    -antigen-update-remote $clone_dir $url
```
Comment (desyncr) [inline bin/antigen.zsh]: I'd put this under a configuration variable. Adding a note on the changelog/release notes about prezto community fork and what means to -update-remote.

File: bin/antigen.zsh [hunk 2]
```diff
@@ -121,9 +121,8 @@ antigen () {
   # used to create the given directory path.
 
   if [[ "$1" == ".zprezto" ]]; then
-    # Prezto's (in `.zprezto`), is assumed to be from `sorin-ionescu`'s
-    # remote.
-    echo https://github.com/sorin-ionescu/prezto.git
+    # Prezto's (in `.zprezto`), is assumed to be from the community fork.
+    echo https://github.com/zsh-users/prezto.git
```
Comment (desyncr) [inline bin/antigen.zsh]: We should move this to a configuration variable, defaulting to `https://github.com/zsh-users/prezto.git`. So they can easily switch between prezto forks, if needed.

File: bin/antigen.zsh [hunk 3]
```diff
@@ -552,7 +565,7 @@ antigen () {
   fi
   export ZDOTDIR=$ADOTDIR/repos/
 
-  antigen-bundle sorin-ionescu/prezto
+  antigen-bundle zsh-users/prezto
```
Comment (desyncr) [inline bin/antigen.zsh]: Here we can use same configuration mentioned above.

File: bin/antigen.zsh (hunk 1 above)
Comment (paulmelnikow) [inline bin/antigen.zsh]: Makes sense to add a configuration variable for the preferred prezto fork.

Are you suggesting here a second variable, which determines whether remotes should be updated for existing clones? If a user decides to switch forks and adjusts configuration appropriately, it doesn't make sense for them to set an additional variable to get antigen to actually install the fork.

**Comment by paulmelnikow** [comment]
Thanks for the review. Makes sense that this should be configurable. Left you a question about the details.

File: bin/antigen.zsh (hunk 1 above)
Comment (desyncr) [inline bin/antigen.zsh]: In the context you mention it doesn't make sense, you're right, I was thinking on current prezto users which will be moved to the community fork.

Now I see it'll create more confusion, with the `prezto repository url` config will be enough.

File: bin/antigen.zsh [hunk 4]
```diff
@@ -117,13 +117,11 @@ antigen () {
...
-  # Takes a repo's clone dir and gives out the repo's original url that was
-  # used to create the given directory path.
+  # Takes a repo's clone dir and unmangles it, to give the repo's original url
+  # that was used to create the given directory path.
 
   if [[ "$1" == ".zprezto" ]]; then
-    # Prezto's (in `.zprezto`), is assumed to be from `sorin-ionescu`'s
-    # remote.
-    echo https://github.com/sorin-ionescu/prezto.git
+    echo "$(cd "$ADOTDIR/repos/.zprezto" && git config --get remote.origin.url)"
```
Comment (paulmelnikow) [inline bin/antigen.zsh]: This is used to generate output for the user. Given this is now configurable, it seems better to check which repo was used.

**Comment by desyncr** [comment]
Merged! Thanks a lot! :+1: @paulmelnikow

**Comment by paulmelnikow** [comment]
For sure! Thanks your help.

**Pull request merged**